package.json

```diff
@@ -2,22 +2,25 @@
   "name": "joi-generate",
   "description": "Generates sample data from Joi schema",
   "version": "0.1.5",
-  "homepage": "https://github.com/Hewlett-Packard-ESS/joi-generate",
+  "homepage": "https://github.com/danlock/joi-generate",
   "author": {
     "name": "Karl Stoney",
     "email": "karl@jambr.co.uk"
   },
+  "contributors": [
+    "Daniel Lockhart"
```

Comment:
Can you add the other contributors, given you have taken their branches and merged into a single PR?